apps/web/src/components/docs/ChildDocs.tsx

```diff
@@ -0,0 +1,8 @@
+import { Docs } from './Docs';
+import { useLoadDocs } from './useLoadDocs';
+
+export const ChildDocs = ({ path }: { path: string }) => {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please add /** */ comment explaining what "ChildDocs" are / how they are different, and how they are used‏